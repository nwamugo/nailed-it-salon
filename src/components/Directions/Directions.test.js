import React from 'react';
import { shallow } from 'enzyme';
import Direction from './index';

describe('ContactSection', () => {
  it('should not have regressed', () => {
    const component = shallow(<Direction />);

    expect(component).toMatchSnapshot();
  });
});
