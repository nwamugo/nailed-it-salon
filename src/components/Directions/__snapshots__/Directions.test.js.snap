// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ContactSection should not have regressed 1`] = `
<div
  className="direction-box"
>
  <Header />
  <hr
    className="direction-container-border-line"
  />
  <div
    className="direction-container"
  >
    <ContactInformation />
    <div
      className="map"
    >
      <withScriptjs(withGoogleMap(Component))
        containerElement={
          <div
            style={
              Object {
                "height": "400px",
              }
            }
          />
        }
        googleMapURL="https://maps.googleapis.com/maps/api/js?key=undefined&v=3.exp&libraries=geometry,drawing,places"
        loadingElement={
          <div
            style={
              Object {
                "height": "100%",
              }
            }
          />
        }
        mapElement={
          <div
            style={
              Object {
                "height": "100%",
              }
            }
          />
        }
      />
      ;
    </div>
  </div>
</div>
`;
